src/js/view/controls/nextuptooltip.js

```diff
@@ -71,13 +75,26 @@ define([
             this._api.next();
         }
 
-        toggle(show) {
+        toggle(show, reason) {
             if (!this.enabled) {
                 return;
             }
             dom.toggleClass(this.container, 'jw-nextup-sticky', !!this.nextUpSticky);
-            dom.toggleClass(this.container, 'jw-nextup-container-visible', show);
-            dom.toggleClass(this._playerElement, 'jw-flag-nextup', show);
+            if (this.shown !== show) {
```

Comment:
If we are showing the UI and don't have nextUp, that is an error somewhere else in the code. The check for nextUp below, is stickly defensive, and intended to avoid from throwing an exception if that occurs.